query-service-impl/build.gradle.kts

```diff
@@ -41,18 +41,38 @@ dependencies {
   implementation("org.hypertrace.core.attribute.service:attribute-projection-registry:0.12.3")
   implementation("org.hypertrace.core.attribute.service:caching-attribute-service-client:0.12.3")
   implementation("com.google.inject:guice:5.0.1")
-  implementation("org.apache.pinot:pinot-java-client:0.6.0") {
+  implementation("org.apache.pinot:pinot-java-client:0.12.0") {
```

Comment:
I think this should be 0.10.0 as we are using Pinot 10 version.
@laxman-traceable @laxmanchekka Is this fine to go with 0.12.0

Reply:
@kotharironak / @saxenakshitiz : Its better keep it in sync with server version (currently, 0.10.0).
This client is used in view-gen-framework as well. Are you planning to upgrade that as well? I vaguely remember there was an issue when we upgraded pinot client version in view-gen-framework. 

@ravisingal : do you recollect the issue? It has something to do when we create a table.

Reply:
Moved to 0.10.0